src/Microsoft.AspNet.Diagnostics.Elm/project.json

```diff
@@ -17,5 +16,6 @@
                 "System.Threading": "4.0.11-beta-*"
             }
         }
-    }
+    },
+    "resource": ["Views/*.cshtml"]
```

Comment:
Resources? Naah, that means they'll get embedded as files in the DLL, which we definitely don't want. I think you just want `compileExclude`: https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/project.json#L116-L122